src/app/components/search/SearchBar.vue

```diff
@@ -1,33 +1,41 @@
 <template>
-  <UnderConstruction>
-    <div class="flex">
-      <label class="hidden" for="search">{{ t('search') }}</label>
-      <input
-        id="search"
-        class="form-input cursor-pointer rounded-r-none dark:border-gray-500 dark:bg-gray-700 dark:placeholder:text-gray-300"
-        :placeholder="t('search')"
-        readonly
-        type="text"
-        @click="navigateToSearch"
-      />
-      <span
-        class="inline-flex cursor-pointer items-center rounded-r-md border border-l-0 border-gray-300 bg-gray-50 px-3 text-sm text-gray-500 dark:border-gray-500 dark:bg-gray-700 dark:text-gray-300"
-        @click="navigateToSearch"
-      >
-        <IHeroiconsMagnifyingGlass />
-      </span>
+  <div class="flex w-full items-center rounded-xl bg-white p-2 px-4 shadow">
+    <div class="mr-4 flex-shrink-0 text-green-800">
+      <IHeroiconsMagnifyingGlass class="h-8 w-8" />
     </div>
-  </UnderConstruction>
+    <input
+      id="search"
+      class="w-full border-none bg-transparent text-xl text-gray-600 placeholder-gray-400 outline-none focus:ring-0 focus:outline-none"
+      :placeholder="t('search')"
+      v-model="localValue"
```

Comment:
Do you have these squiggly lines in your editor?

![image](https://github.com/user-attachments/assets/be55f951-c7a3-4b39-bb4c-7ab2448faec5)